third_party/imgui/BUILD.gn

```diff
@@ -22,18 +27,16 @@ config("imgui_config") {
   ]
   defines = [ "CHIP_IMGUI_ENABLED=1" ]
 
-  # FUTURE: these should be parsed from `sdl2-config --cflags`
+  # Use pkg-config for SDL2 on all platforms
+  configs = [ ":sdl2_pkgconfig" ]
+
```

Comment:
the comment still should remain probably:

- we want flags to be from `sdl2-config --cflags`
- we may want to have a comment on why depending on pkgconfig does not pull  dl and GL libraries

This is better than before for sure, but not yet perfect I imagine, so let's keep the TODO/FUTURE comments

Reply:
updated

Reply:
In fact, the sdl2-config is deprecated and it still exist in sdl2 for legacy reasons.

https://github.com/libsdl-org/SDL/commit/9209942949a2a7c2b117c22cf970e58bf587e109

I believe the future proof solution here is in fact to use dpkg-config (like it is already done).